### ISSUE TITLE
Centralize Vite Configuration Management to Reduce Redundancy

### ISSUE THREAD

**Issue by Leizhenpeng** [Jonghakseo/chrome-extension-boilerplate-react-vite#540]
Currently, there is a considerable amount of duplicated logic across the vite.config.mts files in various page packages under pages.
To streamline our configuration management and reduce complexity at the business level, it's proposed to consolidate these configurations into a single `vite-config` package. 

This approach will not only simplify the maintenance but also align with the goals of issue #524 by centralizing the Vite build files management. 

This change will help in maintaining a cleaner and more scalable structure.

**Comment by PatrykKuniczak** [comment]
@Leizhenpeng That's great idea to create 1 file with base config. are you eager to do it?

**Comment by PatrykKuniczak** [comment]
@Leizhenpeng I was started working on that, but it's a little tricky cause import that vite config seems not that easy, i will try to do it in the near future, but if you're eager let's try, maybe you will do it quicker than me ;)

**Comment by Leizhenpeng** [comment]
You can share your branch and we can try it together haha
@PatrykKuniczak 
> @Leizhenpeng I was started working on that, but it's a little tricky cause import that vite config seems not that easy, i will try to do it in the near future, but if you're eager let's try, maybe you will do it quicker than me ;)

**Comment by PatrykKuniczak** [comment]
@Leizhenpeng Feel free to work https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/559

I can't working on that on next few days, because of the lack of time sadly, you can try to fix it :)
Freaking night shift demolish my entire week, i'm trying to change it :)

**Comment by Leizhenpeng** [comment]
Looking forward to your work, I will keep an eye on it at any time, I tried it with a lot of difficulty haha

**Comment by PatrykKuniczak** [comment]
> Looking forward to your work, I will keep an eye on it at any time, I tried it with a lot of difficulty haha

Yeah, it's really difficult, that's why i give up with .mts and decide to use .mjs only, ts isn't work with everythink.

I hope in the near future new frontend language will be realase, with static types :)

**Comment by PatrykKuniczak** [comment]
Close by #559
